commit dcd8cbed23201416cbd3bbf43f669737693282d7
Author: Paul Eggert <eggert@cs.ucla.edu>
Date:   Wed Mar 1 12:10:57 2023 -0800

    Egypt now uses DST again
    
    From 2023 Egypt plans to observe DST from April's last Friday
    through October's last Thursday (Thanks to Ahmad ElDardiry.)
    Assume 00:00 and 24:00 transitions.
    * NEWS: Mention this.
    * africa (Egypt): Implement this.

diff --git a/NEWS b/NEWS
index f1bc788e..342ad786 100644
--- a/NEWS
+++ b/NEWS
@@ -11,6 +12,10 @@ Unreleased, experimental changes
 
   Changes to future timestamps
 
+    Starting in 2023, Egypt will observe DST from April's last Friday
+    through October's last Thursday.  (Thanks to Ahmad ElDardiry.)
+    Assume the transition times are 00:00 and 24:00, respectively.
+
     Much of Greenland, represented by America/Nuuk, will continue to
     observe DST using European Union rules.  When combined with
     Greenland's decision not to change the clocks in fall 2023,
diff --git a/africa b/africa
index 90018808..24871f3d 100644
--- a/africa
+++ b/africa
@@ -321,6 +321,13 @@ Rule	Egypt	2007	only	-	Sep	Thu>=1	24:00	0	-
 # From Mina Samuel (2016-07-04):
 # Egyptian government took the decision to cancel the DST,
 
+# From Ahmad ElDardiry (2023-03-01):
+# Egypt officially announced today that daylight savings will be
+# applied from last Friday of April to last Thursday of October.
+# https://www.mena.org.eg/ar/news/dbcall/table/textnews/id/10044253
+# From Paul Eggert (2023-03-01):
+# Assume transitions are at 00:00 and 24:00 respectively.
+
 Rule	Egypt	2008	only	-	Aug	lastThu	24:00	0	-
 Rule	Egypt	2009	only	-	Aug	20	24:00	0	-
 Rule	Egypt	2010	only	-	Aug	10	24:00	0	-
@@ -330,6 +337,8 @@ Rule	Egypt	2014	only	-	May	15	24:00	1:00	S
 Rule	Egypt	2014	only	-	Jun	26	24:00	0	-
 Rule	Egypt	2014	only	-	Jul	31	24:00	1:00	S
 Rule	Egypt	2014	only	-	Sep	lastThu	24:00	0	-
+Rule	Egypt	2023	max	-	Apr	lastFri	 0:00	1:00	S
+Rule	Egypt	2023	max	-	Oct	lastThu	24:00	0	-
 
 # Zone	NAME		STDOFF	RULES	FORMAT	[UNTIL]
 Zone	Africa/Cairo	2:05:09 -	LMT	1900 Oct
